api/src/main/java/io/opencensus/stats/AggregationData.java

```diff
@@ -352,6 +409,61 @@ public static DistributionData create(
         Function<? super AggregationData, T> defaultFunction) {
       return p3.apply(this);
     }
+
+    /**
+     * Example points that may be used to annotate aggregated distribution values, associated with a
+     * histogram.
+     *
+     * @since 0.16
+     */
+    @Immutable
+    @AutoValue
+    public abstract static class Exemplar {
+
+      Exemplar() {}
+
+      /**
+       * Returns value of the {@link Exemplar} point.
+       *
+       * @return value of the {@code Exemplar} point.
+       * @since 0.16
+       */
+      public abstract double getValue();
+
+      /**
+       * Returns the observation (sampling) time of the above value.
+       *
+       * @return the observation (sampling) time of the above value.
```

Comment:
The meaning of "sampling" isn't very clear in this Javadoc (and `Exemplar.create`).

Since AggregationData isn't part of the metrics API, I think these exemplars should only be recorded by OpenCensus, so we could just say something like "the time that this Exemplar's value was recorded".

Reply:
The Javadoc for `Exemplar.create` still says "sampling".